snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -39,7 +39,7 @@
         ContainerSnapcraftCmdError,
         SnapdError,
 )
-from snapcraft._options import _get_deb_arch
+from snapcraft.project._arch_mapping import _get_deb_arch
```

Comment:
You're importing a private method from a private module into a completely different part of the project. It implies that perhaps this functionality should live elsewhere.